crates/viewer/re_redap_browser/src/servers.rs

```diff
@@ -130,16 +131,23 @@ impl Server {
                 .ok();
         }))
         .column_blueprint(|desc| {
-            let name = default_display_name_for_column(desc);
-            let name = name
-                .strip_prefix("rerun_")
-                .unwrap_or(name.as_ref())
-                .replace('_', " ");
+            let mut name = default_display_name_for_column(desc);
+
+            if name.starts_with("rerun_") {
+                name = name
+                    .strip_prefix("rerun_")
+                    .unwrap_or(name.as_ref())
+                    .replace('_', " ");
+            }
```

Comment:
nit:
```suggestion
            let name = default_display_name_for_column(desc);
            let name = name.strip_prefix("rerun_").map_or(name, |name| name.replace('_', " "));
```
(or whatever exactly compiles)